config.md

```diff
@@ -7,304 +7,170 @@ This specification outlines the JSON format describing images for use with a con
+     If `group`/`gid` is not specified, the default group and supplementary groups of the given `user`/`uid` in `/etc/passwd` from the container are applied.
+
+   - **Memory** *integer*, OPTIONAL
+
+     Memory limit (in bytes).
+     This acts as a default value to use when the value is not specified when creating a container.
+
+   - **MemorySwap** *integer*, OPTIONAL
+
+     Total memory usage (memory + swap); set to <code>-1</code> to disable swap.
+     This acts as a default value to use when the value is not specified when creating a container.
+
+   - **CpuShares** *integer*, OPTIONAL
+
+     CPU shares (relative weight vs. other containers).
+     This acts as a default value to use when the value is not specified when creating a container.
+
+   - **ExposedPorts** *object*, OPTIONAL
+
+     A set of ports to expose from a container running this image.
+     Its keys can be in the format of:
+`port/tcp`, `port/udp`, `port` with the default protocol being `tcp` if not specified.
+     These values act as defaults and are merged with any specified when creating a container.
+     **NOTE:** This JSON structure value is unusual because it is a direct JSON serialization of the Go type <code>map[string]struct{}</code> and is represented in JSON as an object mapping its keys to an empty object.
+
+   - **Env** *array of strings*, OPTIONAL
+
+     Entries are in the format of <code>VARNAME="var value"</code>.
+     These values act as defaults and are merged with any specified when creating a container.
+
+   - **Entrypoint** *array of strings*
+
+     A list of arguments to use as the command to execute when the container starts.
+     This value acts as a  default and is replaced by an entrypoint specified when creating a container. This field MAY be "null".
+
+   - **Cmd** *array of strings*, OPTIONAL
+
+     Default arguments to the entrypoint of the container.
+     These values act as defaults and are replaced with any specified when creating a container.
+     If an `Entrypoint` value is not specified, then the first entry of the `Cmd` array should be interpreted as the executable to run.
+
+   - **Volumes** *object*, OPTIONAL
+     A set of directories which should be created as data volumes in a container running this image. This field MAY be "null".
+     If a file or folder exists within the image with the same path as a data volume, that file or folder is replaced with the data volume and is never merged. **NOTE:** This JSON structure value is unusual because it is a direct JSON serialization of the Go type <code>map[string]struct{}</code> and is represented in JSON as an object mapping its keys to an empty object.
+
+   - **WorkingDir** *string*, REQUIRED
+
+     Sets the current working directory of the entrypoint process in the container.
+     This value acts as a default and is replaced by a working directory specified when creating a container.
+
+- **rootfs** *object, REQUIRED
```

Comment:
still lack of a `*` at the end of `object`. It looks like
`rootfs *object, REQUIRED`